spec/acceptance/ca_cert_ca_spec.rb

```diff
@@ -2,31 +2,32 @@
 
 case host_inventory['facter']['os']['family']
 when 'Debian'
-  trusted_ca_file_remote = '/usr/local/share/ca-certificates/Globalsign_Org_Intermediate.crt'
-  absent_ca_file_remote = '/etc/pki/ca-trust/source/blacklist/CACert.crt'
+  trusted_ca_file_remote = '/usr/local/share/ca-certificates/DigiCert_G5_TLS_ECC_SHA384_2021_CA1.crt'
```

Comment:
I switch from Globalsign to Digicert certificate here because globalsign certificates are only available for download i DER format.